internal/blockprocessor/committer.go

```diff
@@ -328,24 +332,64 @@ func constructDBEntriesForDataTx(tx *types.DataTx, version *types.Version) []*wo
 	return dbsUpdates
 }
 
-func constructDBEntriesForDBAdminTx(tx *types.DBAdministrationTx, version *types.Version) *worldstate.DBUpdates {
+func constructDBEntriesForDBAdminTx(tx *types.DBAdministrationTx, version *types.Version) ([]*worldstate.DBUpdates, error) {
 	var toCreateDBs []*worldstate.KVWithMetadata
+	var indexForExistingDBs []*worldstate.KVWithMetadata
 
 	for _, dbName := range tx.CreateDBs {
+		var value []byte
+		dbIndex, ok := tx.DBsIndex[dbName]
+		if ok {
+			v, err := json.Marshal(dbIndex.AttributeAndType)
```

Comment:
The use of `dbIndex.GetAttributeAndType()` as below is more idiomatic, no?

Reply:
Done